package_scripts/prepare_ubuntu_pkgs_for_ppa.sh

```diff
@@ -1,20 +1,55 @@
 #!/bin/bash
 # Creates a set of packages for each different Ubuntu distribution, with the
-# intention of uploading them to:
-#   https://launchpad.net/~joseluisblancoc/
+# intention of uploading them to a PPA on launchpad
 #
 # JLBC, 2010
 # [Addition 2012:]
 #
 # You can declare a variable (in the caller shell) with extra flags for the
 # CMake in the final ./configure like:
+#
 #  GTSAM_PKG_CUSTOM_CMAKE_PARAMS="\"-DDISABLE_SSE3=ON\""
 #
 
+
+function show_help {
+    echo "USAGE:"
+    echo ""
+    echo "- to display this help: "
+    echo "prepare_ubuntu_packages_for_ppa.sh -h or -?"
+    echo ""
+    echo "- to package to your PPA: "
+    echo "prepare_ubuntu_packages_for_ppa.sh -e email_of_your_gpg_key"
+    echo ""
+    echo "to pass custom config for GTSAM, set the following"
+    echo "environment variable beforehand: "
+    echo ""
+    echo "GTSAM_PKG_CUSTOM_CMAKE_PARAMS=\"\"-DDISABLE_SSE3=ON\"\""
+    echo ""
+}
+
+while getopts "h?e:" opt; do
+    case "$opt" in
+    h|\?)
+        show_help
+        exit 0
+        ;;
+    e)  packager_email=$OPTARG
+        ;;
+    esac
+done
+
+if [ -z ${packager_email+x} ]; then
+    show_help
+    exit -1
+fi
+
+
 set -e
 
 # List of distributions to create PPA packages for:
-LST_DISTROS=(xenial bionic eoan focal)
+LST_DISTROS=(xenial bionic disco eoan focal)
```

Comment:
disco should be removed, since it's deprecated (EOL) and won't be accepted in PPAs. 
groovy (u20.10) could be added, by the way ;-)

Reply:
I found another bug, introduced at the last minute. I'll close this and will resubmit. Very embarrassing ...